Introduction.md

```diff
@@ -0,0 +1,85 @@
+# Introduction
+
+These standards documents describe a framework for onchain/off-chain stateful bitcoin contracts, relying on real-world observations from third-party, without leaking knowledge of usage.
+Bitcoin contracts are cooperatively built and signed
+by every party involved.
+
+## The Bet: A Short Introduction to Discreet Log Contracts
+
+DLC [were originally proposed by Taddheus Dryja](https://adiabat.github.io/dlc.pdf).
+The goal of DLC is to enable setting up contracts between two parties directly on the Bitcoin blockchain using an oracle to determine the contract outcome.
+
+### Oracle
+
+An oracle is an entity signing messages in reaction to real-world events.
+Before event occurrence the oracle advertises a public nonce that will be used to later produce a signature.
+The public key used for signing must also be known in advance, but need to be unique for each event.
+Once the event has occurred they release a signed message over the event outcome using the previously advertised nonce.
+
+An important feature of DLC is that oracle signatures can be used without an explicit request from the contract participants.
+This characteristic is the reason for the "discreet log" part of the name, as the protocol, depending on its implementation and instantiation, enables parties to hide traces of their contracts to the oracle.
+
+### Trustless execution
+
+A second characteristic of DLCs is that both parties involved in the contract do not need to trust each others.
+The setup and execution of the contract is such that they are guaranteed to always be able to unilaterally close the contract.
+In case of a misbehaving counter-party, trying to close a contract on an outcome for which an oracle signature is not known, the honest protocol participant can claim all the funds locked in the contract.
+Parties are also protected against faulty oracles that would not produce a signature at event occurrence through a refund mechanism
+
+While able to unilaterally close the contracts, parties are expected to generally close the contract cooperatively, as it requires one less transaction to be broadcast.
+
+The current version of the specification does not offer protection against a party colluding with an oracle.
+Mitigation such as using a federation of oracles might be considered in future versions.
+
+## Glossary
+
+### Closing Transaction
+
+A transaction spending from the first output of a [CET](#contract-execution-transaction-cet) unlocking the funds using the appropriate [oracle](#oracle) signature and sending them to an address fully controlled by the broadcasting party.
```

Comment:
> spending from the first output of a CET

Eventually we should not have this be the same every time. 
Shouldn't need to reference to the ordering of the outputs here.

Reply:
Yeah I was a bit lazy on that one. I changed, but not sure if it reads really well, let me know if you have a better idea.

Reply:
LGTM